### ISSUE TITLE
InceptionV3 retrain.py: when frozen graph for inference, different accuracy between DecodeJpeg:0 and DecodeJpeg/contents:0 

### ISSUE THREAD

**Issue by chenyuZha** [tensorflow/models#2076]
### Describe the problem
I use **retrain.py** of model InceptionV3 to train my own datasets. At the end of training, I obtains a frozen graph **retrained_graph.pb** and **retrained_label.txt** . To test with the new datasets, I use the following script to do the inference:[label_image.py](https://github.com/llSourcell/tensorflow_image_classifier/blob/master/src/label_image.py)
In this script, the image is read by its path and the prediction of each image is calculated by 
`image_data = tf.gfile.FastGFile(image_path, 'rb').read()`
`predictions = sess.run(softmax_tensor, 
{'DecodeJpeg/contents:0': image_data})`
 
By use this  inference method, I got 94.7% of accuracy.

Then I found another method to do inference, that  is the image is load as **array**, and then the prediction is calculated by use an other tensor `DecodeJpeg:0`
`image = cv2.imread(image_path)`
`image_array = np.asarray(image)`
`predictions = sess.run(softmax_tensor, 
{'DecodeJpeg:0': image_array})`

By use this inference method, I just got 77% of accuracy. 

Can anybody tell me if I do something wrong ?

**Comment by poxvoculi** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by yanpeilun** [comment]
@chenyuZha hello,i wanna ask you ,the difference of DecodeJpeg:0, and 'DecodeJpeg/contents:0'???
